SpatialGDK/Source/SpatialGDKServices/Public/LocalDeploymentManager.h

```diff
@@ -61,18 +61,14 @@ class FLocalDeploymentManager
 
 	bool bLocalDeploymentRunning;
 	bool bSpatialServiceRunning;
+	bool bSpatialServiceInProjectDirectory;
```

Comment:
Should we set bSpatialServiceInProjectDirectory = false in TryStopSpatialService()?

Reply:
Good idea.